examples/huggingface/neural_speed/run_accuracy.py

```diff
@@ -19,15 +19,15 @@
     parser = argparse.ArgumentParser(description="Evaluate diff for a model")
     parser.add_argument('--model_name', type=str, default="~/Llama-2-7b-chat-hf", help="path to model")
     parser.add_argument('--tasks', type=str, default="lambada_openai")
-    parser.add_argument('--model_format', type=str, default="runtime")
+    parser.add_argument('--model_format', type=str, default="neural_speed")
```

Comment:
please also change example readme

Reply:
I have already checked it and it does not need to be modified.